src/bundle/Resources/public/scss/_tables.scss

```diff
@@ -103,12 +103,6 @@
             padding-left: 0;
         }
 
-        .table &--name {
-            max-width: calculateRem(350px);
-            text-overflow: ellipsis;
-            overflow: hidden;
-        }
```

Comment:
Note: deleted because it seems that it has never worked - checked on eZ Platform 3.4.